WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -248,17 +248,25 @@ def update_events
     end
   end
 
+  def get_nearby_competitions(competition)
+    nearby_competitions = competition.nearby_competitions(Competition::NEARBY_DAYS_WARNING, Competition::NEARBY_DISTANCE_KM_WARNING)[0, 10]
+    nearby_competitions.select!(&:isConfirmed?) unless current_user.can_view_hidden_competitions?
+    nearby_competitions || []
```

Comment:
Why is this needed? How does `nearby_competitions` end up being `nil`? It looks like `def nearby_competitions` [always returns an array](https://github.com/jfly/worldcubeassociation.org/blob/427929eb695edf9fb53a95c59dde7cef1c41f8cd/WcaOnRails/app/models/competition.rb#L567-L572)...